defined variables and functions,

TobyAM opened the pull request:
called the functions and passed the returns to variables which fed more functions

TobyAM committed:
defined variables and functions,
called the functions and passed the returns to variables which fed more functions

matkatmusic reviewed (changes requested):
- Your code produces a ton of warnings when I run it on replit.
- None of your functions match the example or the course coding standard, in terms of formatting.

Please review the instructions as well as the course coding standard and re-request a review when you have made your project conform to the project spec and the coding standard for this course.
- [x] revise

TobyAM committed:
reformatted code blocks {} to meet standards.

included return {}; at the end of functions that need to return something.

removed 'f' at the end of double values.

changed some double types to float.
